common/commands.h

```diff
@@ -186,17 +152,20 @@ namespace commands::scan {
     namespace status {
         static constexpr const char* Subject = "scan-status";
 
+        enum class Status
+        {
+            UNKNOWN,
+            CANCELLED_BY_USER,
+            TERMINATED,
+            IN_PROGRESS
+        };
```

Comment:
See http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps
So, avoid ALL_CAPS naming... because ALL_CAPS for macroses
Why did you decide to rename it?